source/docs/appendices/migration-guides.md

```diff
@@ -72,32 +80,22 @@ The environment variable `FRONT_COMMERCE_PRISMIC_URL` has been removed. Please u
 **ProTip :** You can use the online [express-route-tester@2.0.0](http://forbeslindesay.github.io/express-route-tester) to test your paths.
 </blockquote>
 
-- The method now allows you to register a new route with the prismic client using the `withPrismicRoutes` property. This will allow the Prismic client to resolve the url property for defined documents, and it enables redirection for prismic previews.
-
-```js
-  // Skip prismic route registration
-  PrismicLoader.registerRoutableType({
-    ...
-    typeIdentifier: "album", // document type `album` will resolve to path `/albums/:uid`
-    path: "/album/:uid",     // e.g. `/album/queen`
-    withPrismicRoutes: false // defaults to true
-  })
+- A new `resolvers` property to allow the nested routes content relationship resolution.
 
-  // Register a prismic route with resolvers
+```diff
   PrismicLoader.registerRoutableType({
+    typeIdentifier: "album", // document type "album" will resolve to path `/albums/:uid`
+-   path: "/album/:uid",     // "/album/queen"
++   path: "/:category/:uid", // "/rock-and-roll/queen"
++   resolvers: {
++     category: "category"   // identifier of the Content Relationship in the album Custom Type
++   },
     ...
-    typeIdentifier: "album", // document type `album` will resolve to path `/:category/:uid`
-    path: "/:category/:uid", // e.g. `/rock-and-roll/queen`
-    withPrismicRoutes: {
-      resolvers:{
-        category: "category" // The field name for the content relationship in your Prismic Document
-      }
-    }
   })
 ```
 
 <blockquote class="warning">
-Prismic only supports a depth of 3 levels for the content relationship on route resolvers, please see the [Route Resolver example](https://prismic.io/docs/technologies/route-resolver-nuxtjs#route-resolver-examples) for more information. This issue is also being [tracked](https://community.prismic.io/t/linkresolver-and-nested-paths/96/2) for better support.
+**Depth Limit** The Route Resolver is limited to retrieving data from 3 levels deep, please see the [Route Resolver example](https://prismic.io/docs/technologies/route-resolver-nuxtjs#route-resolver-examples) for more information.
```

Comment:
3 or 2? :angel: https://github.com/front-commerce/developers.front-commerce.com/pull/362#discussion_r832243046

Reply:
Updated to 2

Reply:
Ok, thanks!